packages/manager/modules/octavia-load-balancer/src/onboarding/controller.js

```diff
@@ -25,4 +55,25 @@ export default class OctaviaLoadBalancerOnboardingCtrl {
       type: 'navigation',
     });
   }
+
+  createLoadBalancer($event) {
+    if (this.hasPrivateNetwork) {
+      // TODO: redirect to load balancer creation component
```

Comment:
You can already make the redirection as I already push the creation page on the branch

Reply:
Done!